apps/web/src/app/layout.tsx

```diff
@@ -41,24 +41,7 @@ type RootLayoutProps = {
   children: React.ReactNode;
 };
 
-export const metadata = {
```

Comment:
Denne kan være som før. Siden metadata satt i root layout vil være på alle sider. Så trenger ikke en egen funksjon for det på de andre sidene, de er der allerede.